### ISSUE TITLE
Handle middle names correctly

### ISSUE THREAD

**Issue by stevemoser** [gekitz/GKContactImage#4]
Currently if a name with a first, middle, and last name is supplied the middle name's first letter is used instead of the last name's first letter.

name: "John George Smith"

Expected: "JS"
Actual: "JG"

**Comment by gekitz** [comment]
I agree, that doesn't work properly. The question is, should the library own that problem or should the caller make sure to hand over the "correct" data. Not quite sure about that, what is your opinion?

**Comment by gekitz** [comment]
I agree, that doesn't work properly. The question is, should the library own that problem or should the caller make sure to hand over the "correct" data. Not quite sure about that, what is your opinion?

**Comment by stevemoser** [comment]
I'd say maybe have one method that takes a CNContact, one that tries to make a best guess on one string and another that takes two strings for each of the letters.

Also you could look at what Apple does for inspiration: 
http://apple.stackexchange.com/questions/104894/why-do-some-contacts-on-the-favorites-screen-on-phone-app-show-initials-and-othe
